spreadsheets/intermediate/charts-2/further-customization-1.md

```diff
@@ -0,0 +1,56 @@
+---
+author: Stefan-Stojanovic
+
+aspects:
+  - introduction
+
+type: normal
+
+category: how to
+
+---
+
+# Further customization 1
+
+---
+## Content
+
+Now that we know how to add more data, we can better visualize it by modifying our chart.
+
+The chart from the previous insight:
+
+![previous-chart](https://img.enkipro.com/a8f69242523d86c29148a4c3544b41eb.png)
+
+In the `Chart editor`, we have a `Customize` tab.
+
+**Important note:** The `Customize` tab is different for each chart type.
+
+![customize](https://img.enkipro.com/01638b7e174af98dffa0959033fa9590.png)
+
+Here, we can customize our chart with different styling, change the axis, modify the series, and more.
+
+In the `Chart style` tab we can modify the lines of our line chart and make them smooth.
+
+![smooth](https://img.enkipro.com/ad72c00958bc80dbd6ad5b24958bbeaf.gif)
+
+This tab also allows us to maximize the chart, change the background color, fonts, border color, and plot null values.
+
+Next we have the `Charts & axis titles`. Here we can modify the title and subtitle of the chart, change their fonts, font sizes, colors, and format.
+
+We have made the title bold and added a subtitle:
+
+![change-title](https://img.enkipro.com/a6e2ab6b7cff75d3dca79abb4e3467dd.png)
+
+Next we have the `Series` tab. Here we can edit how the data is shown on the chart, how thick the lines will be, show plot points, add different formating and visual representation per data point, show errors, and more.

```

Comment:
I think an unordered list would be better for this

Reply:
I didn't understand what you mean by this?